lib/config/types.ts

```diff
@@ -409,9 +411,16 @@ export interface PackageRuleInputConfig extends Record<string, unknown> {
   packageRules?: (PackageRule & PackageRuleInputConfig)[];
 }
 
+export interface ConfigMigrationPr {
+  configMigration?: boolean;
+  configMigrationBranch?: string;
+  configMigrationCommitMessage?: string;
+  configMigrationPrTitle?: string;
+}
+
 export interface MigratedConfig {
-  isMigrated: boolean;
-  migratedConfig: RenovateConfig;
+  isMigrated?: boolean;
+  migratedConfig?: RenovateConfig;
```

Comment:
why this?

Reply:
leftover that should have been removed.
now removed